openverse_catalog/dags/providers/provider_api_scripts/brooklyn_museum.py

```diff
@@ -171,16 +170,11 @@ def _get_creators(data):
     return creator
 
 
-def _get_images(image):
-    image_url, thumbnail_url = None, None
+def _get_image_url(image):
     image_url = image.get("largest_derivative_url")
-    if image_url:
-        if "http" not in image_url:
-            image_url = "https://" + image_url
-        thumbnail_url = image.get("thumbnail_url", "")
-        if "http" not in thumbnail_url and thumbnail_url:
-            thumbnail_url = "https://" + thumbnail_url
-    return image_url, thumbnail_url
+    if image_url and "http" not in image_url:
```

Comment:
Should this maybe be a different check?
```suggestion
    if image_url and not image_url.startswith("http"):
```